### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by elviragarciaguzman**
Sebas fight with me now

### CODE REVIEW

File: fibonanci.py [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 
 """
-This code is supposed to calculate the Fibonnaci sequence
+This code is supposed to calculate the FUCK Fibonnaci sequence
 and do some other calculations with it
 """
 
 # FIXME: Refactor the name for this library
-import matplotlib.pyplot as matplotlibpyplotlongnameSebiIhopeyoulikeit
+import matplotlib.pyplot as matplotlibpyplotlongnameSebiIhopeyoulikeit Sebi i'mgoing to kill u
```
Comment (sebastiangeli) [inline fibonanci.py]: Remove the space, or ad a # before "Sebi i'm..."

Comment (sebastiangeli) [reply fibonanci.py]: You can keep the comment in line 5 if you want

**Review by sebastiangeli — CHANGES_REQUESTED** [review]
What the fuck

File: fibonanci.py (hunk 1 above)
Comment (elviragarciaguzman) [reply fibonanci.py]: Maybe I do it...

**Pull request merged**